packages/fiori-elements-writer/test/unit/defaults.test.ts

```diff
@@ -10,20 +10,26 @@ describe('Defaults', () => {
         const template: Template<ALPSettings> = {
             type: TemplateType.AnalyticalListPage,
             settings: {
-                entityConfig: {}
+                entityConfig: {
+                    mainEntityName: ''
```

Comment:
This test case is interesting because it shows that we accept an empty string as a valid entity name. Should we validate this and throw an error, is this something we can ignore and say it is the responsibility of the caller?

Reply:
Im not sure since a consumer of the API may want to generate an app and fill in that detail later maybe (although a placeholder could be provided in that case). Otherwise we could validate for valid entity names (perhaps there are other disallowed values or characters) or document that we do not explicitly validate this and validation is a caller responsibility, as you say.